spec/rspec/mocks/partial_double_spec.rb

```diff
@@ -383,6 +383,22 @@ def defined_private_method
         object.implemented
       end
 
+      context "for a class" do
+        it "only runs the `before_verifying_doubles` callback for the class (not for superclasses)" do
+          subclass = Class.new(klass)
+
+          expect { |probe|
+            RSpec.configuration.mock_with(:rspec) do |config|
+              config.before_verifying_doubles(&probe)
+            end
+
+            allow(subclass).to receive(:new)
+          }.to yield_successive_args(
+            an_object_having_attributes(:target => subclass)
+          )
+        end
+      end
```

Comment:
Since we have not grouped the other specs into states of object instances, and given we only have the one spec, could we flatten this (e.g. remove the `context`)?